regolith/tools.py

```diff
@@ -2104,39 +2105,47 @@ def google_cal_auth_flow():
         token.write(creds.to_json())
     # Save the credentials for the next run
 
-def add_to_gitlab(name):
-    """Takes a newly created event, and adds it to the user's google calendar
+def create_talk_repo(name, rc):
+    """Takes in the presentation name and creates a repo
```

Comment:
add full behavior. ...if repo information is specified in regolithrc.json.

Also, remove "presentation" from the docstring to make the function more reusable